artifactory_test.go

```diff
@@ -5152,3 +5156,74 @@ func readerCloseAndAssert(t *testing.T, reader *content.ContentReader) {
 func readerGetErrorAndAssert(t *testing.T, reader *content.ContentReader) {
 	assert.NoError(t, reader.GetError(), "Couldn't get reader error")
 }
+
+func TestArtifactoryProjectInitMaven(t *testing.T) {
```

Comment:
TestArtifactoryProjectInitMaven --> TestProjectInitMaven
Let's make the same naming format change for the other functions below.